components/Header.jsx

```diff
@@ -0,0 +1,303 @@
+import Link from "next/link"
+import { FormattedMessage } from "react-intl"
+
+import mastodonLogo from "../public/logos/logo-full-purple.svg"
+import Image from "next/image"
+import { useState, useEffect, useRef } from "react"
+import classNames from "classnames"
+import { locales } from "../data/locales"
+import MenuToggle from "./MenuToggle"
+import SVG from "react-inlinesvg"
+
+const useMenu = ({ navigationItems }) => {
+  const [mobileMenuOpen, setMobileMenuOpen] = useState(false)
+  const [primaryMenuItemIndex, setPrimaryMenuItemIndex] = useState(0)
```

Comment:
>when the page loads, the first link is already focused on

9de44ffcb471beb04d34f789f53bc86cf763fe91 should fix!

Reply:
>"Skip to Content" button

not opposed to adding! fwiw it does make sense to me on other sites, esp where

- the headers have a lot of tabstops (9 on github) (10 on creativemornigns)
- they don't use front-end routers, so on every page navigation a user gets dumped back at the first tabstop even if the click came from the nav
  
  there's [a note in the APG doc](https://www.w3.org/WAI/ARIA/apg/example-index/menubar/menubar-navigation.html#:~:text=in%20the%20submenu.-,Otherwise%2C%20activates%20the%20menu%20item%2C%20which%20loads%20new%20content%20and%20places%20focus%20on%20the%20heading%20that%20titles%20the%20content.,-Escape) about moving focus after navigation to the document title (first heading?), which we could do, too

so my hope with the roving tabindex nav was that we wouldn't need a skip link for this header (2 tabstops)